### PR TITLE
Backport debezium-testing module to 1.0.x

### PR DESCRIPTION
**Pull request by jcechace**

### CODE REVIEW

File: debezium-testing/pom.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <artifactId>debezium-parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>debezium-testing</artifactId>
+  <name>Debezium Testing</name>
+  <packaging>pom</packaging>
+  <modules>
+    <module>debezium-testing-testcontainers</module>
```
Comment (jpechane) [inline debezium-testing/pom.xml]: I don't recall, are we going to backport testcontainers too?

File: debezium-testing/pom.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-parent</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
```
Comment (jpechane) [inline debezium-testing/pom.xml]: Should be 1.0?

File: debezium-testing/debezium-testing-openshift/pom.xml [hunk 3]
```diff
@@ -0,0 +1,311 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <groupId>io.debezium</groupId>
+    <artifactId>debezium-testing</artifactId>
+    <version>1.1.0-SNAPSHOT</version>
```
Comment (jpechane) [inline debezium-testing/debezium-testing-openshift/pom.xml]: Should be 1.0?

File: debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/DebeziumContainerTest.java [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    private static Network network = Network.newNetwork();
+
+    private static KafkaContainer kafkaContainer = new KafkaContainer()
+            .withNetwork(network);
+
+    public static PostgreSQLContainer<?> postgresContainer = new PostgreSQLContainer<>("debezium/postgres:11")
+            .withNetwork(network)
+            .withNetworkAliases("postgres");
+
+    public static DebeziumContainer debeziumContainer = new DebeziumContainer("1.1.0.Beta1")
```
Comment (jpechane) [inline debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/DebeziumContainerTest.java]: Should be 1.0?

**Comment by gunnarmorling** [comment]
Yes, we decided to backport TC as it's very helpful for Jakub's work. As
it's a completely new module, it should be easy to do.

File: debezium-testing/pom.xml (hunk 1 above)
Comment (jcechace) [reply debezium-testing/pom.xml]: That was my understanding after talking to @gunnarmorling

File: debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/DebeziumContainerTest.java (hunk 4 above)
Comment (jcechace) [reply debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/DebeziumContainerTest.java]: This one I actually wanted to ask about -- which version? 1.0.1.Final?

Comment (jcechace) [reply debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/DebeziumContainerTest.java]: I actually wanted to ask which version exactly to use. But looking at dockerhub it's clearly 1.0.

**Comment by jpechane** [comment]
@jcechace Applied, thanks

**Pull request merged**